consensus/src/twins_test.rs

```diff
@@ -127,12 +131,18 @@ impl SMRNode {
         }
     }
 
-    fn start_num_nodes(
+    /// Starts a given number of nodes and their twins
+    #[cfg(any(test, feature = "fuzzing"))]
+    fn start_num_nodes_with_twins(
         num_nodes: usize,
+        num_twins: usize,
         playground: &mut NetworkPlayground,
         proposer_type: ConsensusProposerType,
     ) -> Vec<Self> {
-        let ValidatorSwarm { nodes } = generator::validator_swarm_for_testing(num_nodes);
+        assert!(num_nodes >= num_twins);
+
+        let ValidatorSwarm { mut nodes } = generator::validator_swarm_for_testing(num_nodes);
+
```

Comment:
extra line?

Reply:
Removed extra line. In general, are there any style guidelines we follow for lines in code?

Reply:
https://github.com/libra/libra/blob/master/documentation/coding_guidelines.md has some, but consistency per file at the very least is good.